File: app/templates/file-list.py
<?php foreach (glob('fs/*.*') as $f) : $name = str_replace('fs/', '', $f); ?>
for path in paths:
    
  <li class="file-entry" data-fe="{{ file.filename }}">
    <a class="open" href="{{ file.path }}">{{ file.filename }}</a>
    <span class="file-actions">
      <a class="sync" href="{{ file.path }}"></a>
    </span>
  </li>

File: app/static/fs-sync-master/file-list.php
<?php foreach (glob('fs/*.*') as $f) : $name = str_replace('fs/', '', $f); ?>

  <li class="file-entry" data-fe="<?php echo $name; ?>">
    <a class="open" href="<?php echo $f; ?>"><?php echo $name; ?></a>
    <span class="file-actions">
      <a class="sync" href="<?php echo $f; ?>"></a>
    </span>
  </li>

<?php endforeach; ?>
